api/index.ts

```diff
@@ -0,0 +1,83 @@
+import { APIGatewayProxyEvent, Context } from 'aws-lambda'
+import { default as bunyan, default as Logger } from 'bunyan'
+
+import { QuoteHandlerInjector } from '../lib/handlers/quote/injector'
+import { QuoteHandler } from '../lib/handlers/quote/quote'
+import { HemiQuoteQueryParamsJoi, HemiQuoteBodyParams } from '../lib/handlers/quote/schema/quote-schema'
+
+const log: Logger = bunyan.createLogger({
+  name: 'Root',
+  serializers: bunyan.stdSerializers,
+  level: bunyan.INFO,
+})
+
+let quoteHandler: QuoteHandler
+try {
+  const quoteInjectorPromise = new QuoteHandlerInjector('quoteInjector').build()
+  quoteHandler = new QuoteHandler('quote', quoteInjectorPromise)
+} catch (error) {
+  log.fatal({ error }, 'Fatal error')
+  throw error
+}
```

Comment:
Is this `try/catch` needed?

Reply:
I moved it inside the handler, and return a 500 error if anything fails. here: https://github.com/hemilabs/routing-api/pull/1/commits/2115f36ced28fba7ff88616347d7ed5bff3d8c3b